SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/Authentication.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
So this I think is deprecated. This is the old flow for connecting SimPlayers. I think SimPlayers now simply go through the standard flow so we should be ok. This can probably just be removed?

Reply:
Hmm yeah I thought I deleted this. Not sure how it got added back.